mobile-app/app/components/OceanInterface/OceanInterface.tsx

```diff
@@ -149,6 +149,26 @@ export function OceanInterface(): JSX.Element | null {
     return vmmap.output;
   };
 
+  useEffect(() => {
+    const saveTx = async (txId: string) => {
```

Comment:
this will save all tx for all network including testnet and playground

1. Add checking that user is in mainnet
2. Add feature flagging. To be able to disable this via feature flags in case something is wrong with the api